### ISSUE TITLE
Pharokka doesn't look for the good file in meta mode

### ISSUE THREAD

**Issue by rdenise** [gbouras13/pharokka#286]
* pharokka version: 1.4.0
* Python version: 3.10
* Operating System: Ubuntu 22.04.2 LTS

### Description

I try to run pharokka with in meta mode and asking to reorient using dnaapler. But pharokka search for the file dnaapler/dnaapler_reoriented.fasta while the file created is dnaapler/dnaapler_all_reoriented.fasta

```python
Traceback (most recent call last):
  File "/home/remi/miniconda3/envs/pharokka/bin/pharokka.py", line 435, in <module>
    main()
  File "/home/remi/miniconda3/envs/pharokka/bin/pharokka.py", line 155, in main
    shutil.copy(input_fasta, destination_file)
  File "/home/remi/miniconda3/envs/pharokka/lib/python3.10/shutil.py", line 417, in copy
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/home/remi/miniconda3/envs/pharokka/lib/python3.10/shutil.py", line 254, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: 'representatives_38/pharokka/dnaapler/dnaapler_reoriented.fasta'
```

### What I Did

```
pharokka.py -i representatives_38/representatives_38.viridic.fasta -o representatives_38/pharokka -d databases/Pharokka/20230823 -t 20 -m -s --dnaapler --meta_hmm
```

**Comment by gbouras13** [comment]
Hi @rdenise,

thanks for finding this bug - I’ll patch it and release a fixed updated version in the next few days.

George

**Comment by gbouras13** [comment]
Hi @rdenise,

This has been fixed in v1.4.1. Please update and give it another go :)

George

**Comment by rdenise** [comment]
Thank you!
